### DISCUSSION TITLE
access gocryptfs from multiple servers

### DISCUSSION THREAD

**Discussion by jumoog** [rfjakob/gocryptfs#825] [Q&A] [answered]
Is it possible to use the same gocryptfs at the same time on different servers? I have gocryptfs files on a NAS. If that is not possible, it would help to mount a system as read only only?

**Comment by rfjakob** [comment, marked as answer]
Yes, possible. Use the [-sharedstorage](https://github.com/rfjakob/gocryptfs/blob/master/Documentation/MANPAGE.md#-sharedstorage) flag when mounting.
